diff --git a/types/index.d.ts b/types/index.d.ts
index b459dfc9028667014cd0f7c575c54f9b1bdeff0e..e19bc317c4f0c3222d1a1ee69f0b22ecd693b3c7 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -6,7 +6,7 @@ import {
 } from '@rspack/core';
 import * as https from 'https';
 
-export = Config;
+export default Config;
 
 // The compiler type of Rspack / webpack are mismatch,
 // so we use a loose type here to allow using webpack plugins.
